### ISSUE TITLE
CC-By is not suitable for music production related content

### ISSUE THREAD

**Issue by falkTX** [geekosdawproject/libre-vitalium-presets#2]
Please do not recommend the use of cc-by.
Tracking the usage of such things in order to give attribution is a very big pain, plus if someone wants to sell Vitalium (completely fine and legal under GPL) there is this weird restriction on top that certain content needs to have attribution.

Only CC-0 is good for this.

See the effort by Nils that describes the situation much better than I can https://hilbricht.net/foss-sampled-instruments.html

**Comment by fpesari** [comment]
Hello @falkTX,

while I agree with you on the fact that attribution in presets is very hard, I don't feel like refusing presets released under perfectly fine libre licenses. Contributors are free to choose CC0 or CC-By or even CC-By-SA but I can't be the one who refuses them their right to choose how their work can be used.

The best I can do is link to Nils' article in the README and organize presets under three different directories: cc0, cc-by and cc-by-sa, so that people selling Vitalium can simply ship the cc0 samples and ignore the other ones.

**Comment by falkTX** [comment]
they are perfectly fine licenses yes, but their not suitable to the point at hand.
GPL is a fine license but we should not use it for artwork, for example.

Accepting CC-by patches basically makes the project "tainted" where one would then need to go through each preset to know if they are usable for them or not. It is just very tedious and inspiration-breaking.

Basically releasing patches as CC-by is asking any music/sound produced with it to also be released as CC-by (plus attribution), I think that is a major issue that will make many just ignore the project.
You can think this is actually a good thing of course, in which case carry on then.

**Comment by fpesari** [comment]
I get your point, Filipe, but personally I have no issue with the requirements of CC-By and CC-By-SA, especially because we are clearly marking which presets are released under which license. There is zero risk of confusion; if someone downloads everything without reading the license terms then they are at fault, not us who are merely doing our best to keep everything 100% free and legal.

If someone makes a good patch but refuses to release it under CC0 because they want attribution, should we refuse it? I think that would be a disservice to the community.

**Comment by falkTX** [comment]
All is fine, but just to make sure you understand...

> If someone makes a good patch but refuses to release it under CC0 because they want attribution, should we refuse it?

You know this means that music, sound or any other content produced using this patch needs to give attribution to said patch right?
Do you imagine people giving attribution to every single wavetable and preset, samples, etc?
Really don't see how that scales, but that could be just me.

**Comment by fpesari** [comment]
But you know what, I have doubts about the fact that presets can be copyrighted, to be honest. Wavetables are samples and certainly can be copyrighted (but a case could be made for basic waves. Can a mathematical function be copyrighted?), but can presets be copyrighted? 

Of course, you can't legally distribute something like DX7 factory presets, they belong to Yamaha. But the sounds that comes out by the synth playing that preset, as far as I know, if it is not based on sampling is not really copyrighted. That's why you can find free 808 samples around but not LinnDrum ones. 

Now that I think of it, I think the current requirements don't tackle a legal issue which is very hard to figure out, especially since we are not lawyers. I think we are definitely going to reject all wavetables which are not original or CC0, that's for sure.

As for presets, if you can find something about the aforementioned issue (it seems always legal to sample the output of non-sample based synths without attribution), perhaps we can talk about making them CC0 too just to avoid requirements which can't be legally enforced.

**Comment by fpesari** [comment]
So what I did not say explicitly is: it is possible that putting wavetable issues aside (let's say they are all original or CC0), the license terms of the presets could apply to the presets itself (as in the actual data files) but not the output of the synth playing that preset. So, even if a presets were CC-By or CC-By-SA (or even proprietary), the output of the synth playing that patch would be considered a legally separate thing, just like you can edit a file using Emacs without it being bound by the terms of the GPLv3+.

**Comment by falkTX** [comment]
Well I am not a lawyer and neither are you, going into a grey area that has not yet been tested in courts is something I bet musicians do not want to adventure themselves in.
It sounds reasonable to be as you say, but are we confidently certain of it?

The push for CC0 is to basically just avoid any *potential* issues altogether.

**Comment by unfa** [comment]
I would agree that all stock presets should be CC-0. When you buy a hardware synth or a proprietary virtual one you don't have to keep track what presets you used and who needs credit. Nobody expects something like this to be required of them, and no hardly one would attribute the presets anyway. Enforcing CC-0 sets the expectations right to the preset authors, and provides an expected situation to users. I think the preset authors should be credited in the synthesizer, but not require attribution on music made with their patches.

If a user installs an extra preset pack licensed under CC-BY or whatever, they take ful responsibility of making things hard on themselves or violating the licenses. I say - let's not embed a legal minefield into an open-source synth.

**Comment by fpesari** [comment]
Done; please someone, check the README now to say if it is OK or not.

Also @unfa, what about running a contest for making CC0-licensed presets and wavetables? Users don't have to use this repo, we can collect them later as long as there is clear licensing info.
